Activities/Database/Database.build.props

```diff
@@ -2,6 +2,6 @@
   <PropertyGroup>
     <Company>UiPath</Company>
     <Copyright>© UiPath</Copyright>
-    <VersionPrefix>1.5.0</VersionPrefix>
+    <VersionPrefix>1.6.0</VersionPrefix>
```

Comment:
Why are these changed here ?
Same for other package versions.

Reply:
renamed